sphinx/search/ja.py

```diff
@@ -13,16 +13,16 @@
 import os
 import re
 import sys
-from typing import TYPE_CHECKING, Any, Dict, List
+from typing import Any
 
 try:
-    import MeCab
+    import MeCab  # type: ignore[import-not-found]
     native_module = True
 except ImportError:
     native_module = False
 
 try:
-    import janome.tokenizer
+    import janome.tokenizer  # type: ignore[import-not-found]
     janome_module = True
```

Comment:
the capitalisation of `*_module` variables is inconsistent. Since these are constants they should probably be all-caps

Reply:
I don't think this should be part of this PR.